tools/generate-changelog.py

```diff
@@ -182,30 +196,47 @@ def read_remote_git_url() -> str:
     return remote_url.removesuffix(".git")
 
 
+def log_between_commits(start, end):
+    return subprocess.check_output(['git', 'log', f'{start}..{end}']).decode()
```

Comment:
I think we should take only the title of the commit, having the full commit information is too noisy I think

Reply:
How about we use `git shortlog` instead?

Reply:
Now it looks like this: https://paste.ubuntu.com/p/qcyYNVwMXz/

Reply:
Agreed, something we need to be aware of when merging, that commits look nice